frontend/src/metabase/query_builder/components/view/ViewFooter/LeftViewFooterButtonGroup.module.css

```diff
@@ -0,0 +1,10 @@
+.Root {
+  .FooterButtonGroup {
+    display: inline-flex;
+    align-items: stretch;
+
+    .Button {
```

Comment:
Is all this nesting necessary?

Reply:
the answer is usually yes because of specificity: we have to deal with emotion from mantine and emotion is always after css modules, so emotion wins with the same specificity